docs/distribution.md

```diff
@@ -0,0 +1,19 @@
+# Project
+
+## Contents
+-[blah](architecture.md)
```

Comment:
Yes, this is something I need for the Mkdocs tooling. They require a specific format.